src/AppBundle/DataFixtures/ORM/bluray.yml

```diff
@@ -5,4 +5,4 @@ AppBundle\Entity\Bluray:
         type: <word()>
         trailer: Z6Z1f4Lt01o
         available: 1
-        picture: <picture()>
+        picture: <picture()>
```

Comment:
It's recommended to keep a blank line at the end of each file (see http://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-file)

See also https://github.com/FriendsOfPHP/PHP-CS-Fixer to automatically fix your code.